src/services/getATFListService.js

```diff
@@ -5,7 +5,7 @@ const path = require('path')
 const getATFList = () => {
   return new Promise((resolve, reject) => {
     try {
-      resolve(JSON.parse(fs.readFileSync(path.resolve(__dirname, '../mocks/mock-atf.json'), 'utf8')))
+      resolve(JSON.parse(fs.readFileSync(path.resolve(__dirname, '../../tests/resources/mock-atf.json'), 'utf8')))
```

Comment:
we would be changing scope of this service from being a stub to a facade service, hence expect another ticket which would retrieve this data from a DB or S3 bucket. For now this is okay